proof/refine/ARM/IpcCancel_R.thy

```diff
@@ -2660,6 +2660,106 @@ proof -
   done
 qed
 
+lemma replyUnlink_valid_tcb_state'_restart[wp]:
```

Comment:
Is this lemma name what it is supposed to be? I don’t quite understand it but maybe I am missing something.

Reply:
Will change this to
```suggestion
lemma replyUnlink_sch_act[wp]:
```